quilt/tools/command.py

```diff
@@ -372,41 +381,92 @@ def _push_package(dry_run=False):
             }
         )
 
-    _push_package(dry_run=True)
+    print("Fetching upload URLs from the registry...")
```

Comment:
Yeah - in case of dogscats, it's like 8MB of URLs. (Compressed, but could still take a while...)